ui/app/api/peers/create/route.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  CreatePeerRequest,
+  CreatePeerResponse,
+  CreatePeerStatus,
+} from '@/grpc_generated/route';
+import { GetFlowServiceClient } from '@/rpc/rpc';
+import { constructPeer } from '../util';
```

Comment:
lets move this method to this class, there isn't much happening in the utils still.